apps/portal/src/content/docs/components/checkbox.mdx

```diff
@@ -78,6 +80,25 @@ import { DocsPage } from "@/components/docs-page";
             onCheckedChange={setCheckedWOLabel}
           />
         </div>
+
+        {/* Error state */}
+        <div className="space-y-2">
+          <h4 className="text-sm font-medium">Error state</h4>
+          <div className="flex items-center space-x-8">
+            <Checkbox
```

Comment:
we should document error status with the Form Checkbox Group component, so that it shows the error message, and not just the red highlight on the checkbox